Kitodo/src/main/java/org/kitodo/production/forms/ProcessListBaseView.java

```diff
@@ -453,4 +454,20 @@ public void delete(ProcessDTO processDTO) {
     public DeleteProcessDialog getDeleteProcessDialog() {
         return this.deleteProcessDialog;
     }
+
+    /**
+     * Check and return whether process with given ID has an empty generator folder or not.
+     *
+     * @param processId process ID
+     * @return whether given URI points to empty directory or not
+     */
+    public boolean hasImages(int processId) {
+        try {
+            return FileService.hasImages(processId);
```

Comment:
At the moment, we do access File Service using the Service Loader. However, I prefer static access where possible. See #3327.

Reply:
So I take it you agree with `hasImages` remaining `static`?

Reply:
@Kathrin-Huber should decide